package.json

```diff
@@ -3,7 +3,11 @@
   "description": "Hoodie Client Core",
   "main": "index.js",
   "scripts": {
-    "test": "standard && npm run -s test:node | tap-spec",
+    "build": "npm-run-all -p build:*",
+    "build:dev": "browserify -s Hoodie index.js -o hoodie.js",
+    "build:min": "browserify -s Hoodie -g uglifyify index.js | uglifyjs -c -o hoodie.min.js",
```

Comment:
Open for change here. Seemed simplest at first. 

Whatever we go with this is going to be part of the public API and breaks hoodie-server if we change it.